src/simplewallet/simplewallet.cpp

```diff
@@ -101,6 +101,7 @@ enum TransferType {
 
 namespace
 {
+  const auto allowed_priority_strings = {"default", "unimportant", "normal", "elevated", "priority"};
```

Comment:
The current CLI wallet lists all four priorities even though v5 fork hasn't happened yet. Do we want to add some logic when showing the description texts on the CLI side too?